onnxruntime/core/mlas/lib/mlasi.h

```diff
@@ -93,6 +93,8 @@ Module Name:
 
 #define MLAS_UNREFERENCED_PARAMETER(parameter) ((void)(parameter))
 
+#include "core/common/common.h"
```

Comment:
We have been trying to keep MLAS as independent as possible and we hanging on at least for now :-) This would destroy all these efforts.  Is there anything there absolutely needed?

Reply:
Implement MLAS_THROW_EX for independence instead.